common/src/webida/plugins/workspace/Node.js

```diff
@@ -1119,7 +1119,11 @@ define(['require',
 
     Node.compare = function nodeCompare(a, b) {
         if (a.isInternal === b.isInternal) {
-            if (a.name < b.name) {
+            if (a.name.toLowerCase() < b.name.toLowerCase()) {
```

Comment:
How about using vars to avoid function call repetition?
````js
    Node.compare = function nodeCompare(a, b) {
        var lowA, lowB;
        if (a.isInternal === b.isInternal) {
            lowA = a.name.toLowerCase();
            lowB = b.name.toLowerCase();
            if (lowA < lowB) {
                return -1;
            } else if (lowA > lowB) {
                return 1;
            } else if (a.name < b.name) {
                return -1;
            } else if (a.name > b.name) {
                return 1;
            } else {
                return 0;
            }
        } else {
            if (a.isInternal) { // directory node precedes a file node.
                return -1;
            } else {
                return 1;
            }
        }
    };
````